cmd/signerparams.go

```diff
@@ -42,23 +42,23 @@ func (p *SignerParams) Edit(ctx ActionCtx) error {
 	var err error
 
 	sctx := ctx.StoreCtx()
-	p.signerKP, err = sctx.ResolveKey(p.kind, KeyPathFlag)
-	if err != nil {
-		return err
-	}
+	p.signerKP, _ = sctx.ResolveKey(p.kind, KeyPathFlag)
 
 	// skip showing a signer
 	if p.signerKP != nil && ctx.StoreCtx().Store.IsManaged() {
 		return nil
 	}
-	ks := sctx.KeyStore
-	switch p.kind {
-	case nkeys.PrefixByteOperator:
-		KeyPathFlag = ks.GetOperatorKeyPath(sctx.Operator.Name)
-	case nkeys.PrefixByteAccount:
-		KeyPathFlag = ks.GetAccountKeyPath(sctx.Account.Name)
-	case nkeys.PrefixByteCluster:
-		KeyPathFlag = ks.GetClusterKeyPath(sctx.Cluster.Name)
+
+	if KeyPathFlag == "" {
```

Comment:
If this overrides others maybe do this first and return so its clearer what we are doing?

Reply:
It cannot some other operation could have set the default - (setting the account name via a flag, etc). So this check is delayed until the last possible moment during interactive.

Reply:
But flags override everything correct?